edison/src/test/java/edu/illinois/cs/cogcomp/edison/features/factory/TestWordFeatureFactory.java

```diff
@@ -191,17 +191,17 @@ public final void testFeatureCollection() throws Exception {
             }
         }
     }
-
-    public final void testGazetteerFeatures() throws Exception {
-        System.out.println("\tTesting gazetteer features");
-        WordFeatureExtractor fex =
-                WordFeatureExtractorFactory.getGazetteerFeatureExtractor("gazetteer",
-                        new GazetteerViewGenerator("gazetteers", ViewNames.GAZETTEER));
-
-        for (TextAnnotation ta : tas) {
-            runTest(ta, fex);
-        }
-    }
+// Not needed anymore, as test of SimpleGazetteerAnnotator in SimpleGazetteerAnnotatorTest.java suffice.
```

Comment:
I think they are testing slightly different things. The test for `SimpleGazetteerAnnotatorTest ` is testing its functionality directly. While this is testing correct functionality of `SimpleGazetteerAnnotatorTest` inside `WordFeatureExtractorFactory`. 
What do you think?